apps/backoffice-v2/src/domains/documents/hooks/useDocumentsAdapter/useDocumentsAdapter.tsx

```diff
@@ -123,7 +122,14 @@ export const useDocumentsAdapter = ({
     }));
   };
 
-  const documents = getDocuments();
+  const documents = useMemo(getDocuments, [
```

Comment:
I think might as well either wrap getDocuments in useCallback or inline the logic into the useMemo?